packages/aws-cdk/lib/cli/user-configuration.ts

```diff
@@ -305,8 +305,9 @@ export function commandLineArgumentsToSettings(argv: Arguments): Settings {
     ignoreNoStacks: argv['ignore-no-stacks'],
     hotswap: {
       ecs: {
-        minimumEcsHealthyPercent: argv.minimumEcsHealthyPercent,
-        maximumEcsHealthyPercent: argv.maximumEcsHealthyPercent,
+        minimumHealthyPercent: argv.hotswapEcsMinimumHealthyPercent,
+        maximumHealthyPercent: argv.hotswapEcsMaximumHealthyPercent,
```

Comment:
So actually a user could never override this behavior from the CLI. Which means, we can change the argument names to be more descriptive:

- `hotswap-ecs-minimum-healthy-percent`
- `hotswap-ecs-maximum-healthy-percent`

These names are derived and match the argument names that I think we initially wanted, based on:

https://github.com/aws/aws-cdk-cli/blob/382393730b6e8870cfb5d22aaf40c51b0b73b887/packages/%40aws-cdk/toolkit-lib/lib/api/hotswap/common.ts#L91-L96